src/components/CodeEditorWindow/index.js

```diff
@@ -49,7 +49,9 @@ const CodeEditorWindow = ({ onChange, code, onChangeResult }) => {
     )
 
     // Register Code Lens Provider (Run Button)
+    if(runBtnCommandId==="DYNAMIC_1"){
```

Comment:
unstable solution, there is no guarantee that `runBtnCommandId` will always be `DYNAMIC_1`